tests/integration/test_web_client.py

```diff
@@ -219,6 +219,26 @@ def test_creation_lifecycle(self, antares_web: AntaresWebDesktop):
         assert actual_hydro.area_id == area_fr.id
         assert actual_hydro.properties == area_fr.hydro.properties
 
+        # tests study reading method and comparing ids, name, areas and settings
+        actual_study = read_study_api(api_config, study.service.study_id)
+
+        assert study.service.study_id == actual_study.service.study_id
+        assert study.name == actual_study.name
+        assert study.version == actual_study.version
+        assert list(study.get_areas().keys()) == list(actual_study.get_areas().keys())
+        assert (
```

Comment:
It's better. You can introduce 
`actual_area_fr = actual_study.get_areas()["fr"]` and `expected_area_fr = study.get_areas()["fr"]` to avoid calling get_areas multiple times.
Also I would prefer you do the same check as the ones for areas (line 228) for thermal, storages and renewables as it would check more things.
Last thing, I believe you can also check the settings